x-pack/plugins/observability_solution/slo/public/pages/slo_details/components/header_title.tsx

```diff
@@ -19,19 +19,14 @@ export interface Props {
   showTitle?: boolean;
```

Comment:
@shahzad31 Looks like we don't use this prop `showTitle` anymore. Can you remove this?